packages/components/src/components/link/component.tsx

```diff
@@ -91,6 +91,8 @@ export class KolLinkWc implements KoliBriLinkAPI {
 
 	public render(): JSX.Element {
 		const { isExternal, tagAttrs, goToProps } = this.getRenderValues();
+		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+		const label = (typeof this._label === 'string' && this._label.length >= 3 ? this.state._label : this.state._href)!;
```

Comment:
Diese Condition muss anders formuliert werden, weil die eigentlich Validierung von Label erfolgt im validateLabel. Ja vielleicht ist hier dann eine Prüfung auf this._label sinnvoll.

Reply:
this.state._label ist immer mindestens 3 Zeichen lang, da der Standard '...' ist und nur wenn >= 3 Zeichen ankommen wirds überschrieben.